cmd/xl-sets.go

```diff
@@ -1668,6 +1668,6 @@ func (s *xlSets) HealObjects(ctx context.Context, bucket, prefix string, healObj
 	return nil
 }
 
-func (s *xlSets) ListObjectsHeal(ctx context.Context, bucket, prefix, marker, delimiter string, maxKeys int) (loi ListObjectsInfo, err error) {
+func (s *xlSets) ListObjectsAll(ctx context.Context, bucket, prefix, marker, delimiter string, maxKeys int) (loi ListObjectsInfo, err error) {
 	return s.listObjects(ctx, bucket, prefix, marker, delimiter, maxKeys, true)
```

Comment:
since we set the heal as true here changing this function can complicate things, we should choose an appropriate name.

Reply:
> since we set the heal as true here changing this function can complicate things, we should choose an appropriate name.

Probably ListObjectsNoQuorum ?

Reply:
Yes that is better..

Reply:
> Yes that is better..

Done